spyderlib/config.py

```diff
@@ -528,7 +528,6 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
-              'editor/print': "Ctrl+P",
               'editor/close all': "Ctrl+Shift+W",

```

Comment:
`Print` is not an action that is constantly needed in IDEs, so I'm fine with removing the shortcut for it.

In any case, it'll always be available from the `File` menu ;-)